docs/index.md

```diff
@@ -8,7 +8,7 @@ import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vu
 
 # 👨‍⚖️ `jstz`
 
-`jstz` (pronouced 'justice') is a JavaScript server runtime for Tezos 2.0
+`jstz` (pronounced 'justice') is a JavaScript server runtime powered by Tezos Smart Optimistic Rollups
```

Comment:
```suggestion
`jstz` is a JavaScript server runtime powered by Tezos Smart Optimistic Rollups
```

Reply:
I disagree with this change

Reply:
We should still aim to give a pronunciation